tools/cli/internal/apiversion/version.go

```diff
@@ -67,10 +70,31 @@ func WithDate(date time.Time) Option {
 	return func(v *APIVersion) error {
 		v.version = date.Format(dateFormat)
 		v.versionDate = date
+		v.stabilityVersion = StableStabilityLevel
 		return nil
 	}
 }
 
+// WithStabilityLevel sets the version and stability level on the APIVersion.
+func WithStabilityLevel(version, stabilityLevel string) Option {
```

Comment:
```suggestion
func WithStabilityLevel(stabilityLevel string) Option {
```
chain the options don't make the options make multiple thigs

Reply:
forgot to remove this piece of code as it is not used, I will remove it. Good catch 🤙